ports/servo/build.rs

```diff
@@ -83,3 +89,10 @@ fn android_main() {
     println!("cargo:rustc-link-lib=log");
     println!("cargo:rustc-link-lib=android");
 }
+
+fn windows_main() {
+    let mut res = winres::WindowsResource::new();
+    res.set_resource_file("windows.rc");
```

Comment:
@Coder206 I have experienced similar icon corruption issues without this PR applied (see #16326).